wallet/src/de/schildbach/wallet/ui/main/WalletActivity.java

```diff
@@ -320,129 +195,6 @@ protected void error(Exception x, final int messageResId, final Object... messag
         }
     }
 
-    @Override
-    public void onRequestPermissionsResult(final int requestCode, final String[] permissions,
-                                           final int[] grantResults) {
-        super.onRequestPermissionsResult(requestCode, permissions, grantResults);
-        if (requestCode == REQUEST_CODE_BACKUP_WALLET) {
-            if (grantResults.length > 0 && grantResults[0] == PackageManager.PERMISSION_GRANTED)
-                showBackupWalletDialog = true;
-            else
-                showDialog(DIALOG_BACKUP_WALLET_PERMISSION);
-        } else if (requestCode == REQUEST_CODE_RESTORE_WALLET) {
-            if (grantResults.length > 0 && grantResults[0] == PackageManager.PERMISSION_GRANTED)
-                handleRestoreWallet();
-            else
-                showDialog(DIALOG_RESTORE_WALLET_PERMISSION);
```

Comment:
The code in this method seems to be unreachable. Do we use the backup and restore from backup dialogs on the home screen? Looks like it was invoked from the context menu which we don't have now.

Reply:
You are right.  Backup and Restore were handled by menus and the home screen with version 6, but that was removed in version 7.  We no longer need this code.